lib/active_model/serializer.rb

```diff
@@ -476,9 +479,7 @@ def scope
 
     def _serializable_hash
       return nil if @object.nil?
-      @node = attributes
```

Comment:
`@node` needs to be assigned outside of `_serializable_hash`, since associations can differ based on the serializer used, and the `_serializable_hash` method itself is cached inside `serializable_hash`